storybook/.babelrc

```diff
@@ -1,6 +1,7 @@
 {
 	"presets": [ "@wordpress/babel-preset-default" ],
 	"plugins": [
+    "babel-plugin-emotion",
```

Comment:
Nit: whitespaces, I don't think we lint this type of files :(

Reply:
Oh! My bad. Fixing now. Thank you for catching <3